charts/datadog/ci/agent-with-lifecycle-handler.yaml

```
@@ -0,0 +1,7 @@
+datadog:
+  apiKey: "00000000000000000000000000000000"
+agents:
+  enabled: true
+  lifecycle:
+    sleep:
+      seconds: 70
```

Should the `sleep` block be under `lifecycle.[preStart|postStart]`? https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.30/#lifecycle-v1-core

I get an `unknown field` warning testing this file in GKE 1.31: `unknown field "spec.template.spec.containers[0].lifecycle.sleep"`

```suggestion
  lifecycle:
    postStart:
      sleep:
        seconds: 70
```

---

fixed!